packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -44,6 +44,23 @@ function Sidebar(): JSX.Element {
 
   const logoLink = navigationPlugin.getURL('hub', '#/');
 
+  const shouldHideElement = (node: Node) => {
+    if (betaVersion === 2) {
+      if (node.id === 'services') return false;
+      if (node.count === false) return false;
```

Comment:
count is a boolean not a number ? That's confusing

Reply:
Hey @marie-j 

It's actually both. If you take a look at the `node.ts` file, you will see that it's an optionnal attribute, that can either be a boolean or a number.
 
I don't recall why it was both, but i think that it's related to the json config of services we have, we write "false" to avoid showing the element before starting any count or something like that.